frontend/src/components/pages/CampCreation/CampCreationFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Button, Flex, Spacer } from "@chakra-ui/react";
+import CampCreationPages from "./CampCreationPages";
+
+export type CampCreationFooterProps = {
+  currentStep: CampCreationPages;
+  isCurrentStepCompleted: boolean;
+  handleStepNavigation: (stepsToMove: number) => void;
+};
+
+const CampCreationFooter = ({
+  currentStep,
+  isCurrentStepCompleted,
+  handleStepNavigation,
+}: CampCreationFooterProps): React.ReactElement => {
+  const onNextStep = () => {
+    if (isCurrentStepCompleted) {
+      handleStepNavigation(1);
+    }
+    if (currentStep === CampCreationPages.RegistrationFormPage) {
+      // eslint-disable-next-line no-console
+      console.log("Publish camp."); // Temporary.
+    }
+  };
+
+  return (
+    <Flex
+      bg="white"
+      boxShadow="sm"
+      minH="92px"
+      width="100vw"
+      align="center"
+      justify={{ sm: "center", lg: "end" }}
```

Comment:
Did you happen to get this from Bowen's PR? If so, we don't need to make the staff tool responsive so I think we can probs remove this?

Reply:
Done.